include/OpenColorIO/OpenColorTransforms.h

```diff
@@ -749,7 +749,100 @@ OCIO_NAMESPACE_ENTER
     
     //!cpp:function::
     extern OCIOEXPORT std::ostream& operator<< (std::ostream&, const MatrixTransform&);
+
+    //!rst:: //////////////////////////////////////////////////////////////////
+
+    //!cpp:class:: Represents a range transform
+    //
+    // The Range is used to apply an affine transform (scale & offset) and
+    // clamps values to min/max bounds on all color components except the alpha.
+    // The scale and offset values are computed from the input and output bounds.
+    // 
+    // .. note::
+    //    Refer to section 7.2.4 in specification S-2014-006
+    //    "A Common File Format for Look-Up Tables" from the
+    //    Academy of Motion Picture Arts and Sciences and 
+    //    the American Society of Cinematographers.
+    // 
+    // .. note::
```

Comment:
Patrick and I discussed this Michael and I think you raise a valid point that it could be convenient to have the noClamp option available to config authors.  This would essentially mirror what is in the CLF.  Under the hood the RangeTransform would then decide whether to emit a RangeOp (that always clamps) or a MatrixOp (that obviously does not clamp).

Reply:
It unfortunately implies more changes than anticipated so I plan to do it in a dedicated pull request.